src/data_info/clients/notifications.clj

```diff
@@ -42,11 +42,18 @@
     user "rename" end-paths))
 
 (defn trash-notification
-  [user start-paths failed?]
+  [user start-paths end-paths failed?]
   (amend-notification
     {:subject (str (completed-text failed?) " moving " (count start-paths) " files(s)/folder(s) to trash")
-     :message (str (completed-text failed?) " moving " (string/join ", " start-paths) " to trash")}
-    user "trash" [(paths/user-trash-path user)]))
+     :message (str (completed-text failed?) " moving " (string/join ", " start-paths) " to trash at " (string/join ", " end-paths))}
+    user "trash" end-paths))
+
+(defn delete-notification
+  [user start-paths failed?]
+  (amend-notification
+    {:subject (str (completed-text failed?) " deleting " (count start-paths) " file(s)/folder(s)")
+     :message (str (completed-text failed?) " deleting " (string/join ", " start-paths))}
+    user "delete" [(paths/user-trash-path user)]))
```

Comment:
This new notification type is close to the old trash one in that it passes the trash, but it doesn't talk about moving (since it doesn't involve any)